cfpb/ccdb5-ui
# DATAP-1583 - company received date filter fixes

fixing company received date filter

This filter was not clearing when you attempt to clear it, either by clicking Clear in the date picker, or clicking "Clear all filters"

## Additions

- added action to clear the filter

## Removals

- removed some convoluted logic where we were trying to be clever and reuse the same action for different filters

## Changes

-

## Testing

- yarn start
- Add dates to the "The date the CFPB sent the complaint to the company" filter
- see dates get applied

<img width="395" alt="Screenshot 2024-11-01 at 2 44 42 PM" src="https://github.com/user-attachments/assets/c4cd4fba-f398-422a-9cf8-f53720a3ff16">

- Click Clear in each date to confirm the date can be cleared

Also Click "Clear all filters" to clear out the Company received date filter



## Screenshots


## Notes

-

## Todos

-

## Checklist
